src/fragments/lib/auth/js/start.mdx

```diff
@@ -127,6 +127,9 @@ Amplify.configure({
 // You can get the current config object
 const currentConfig = Auth.configure();
 ```
+<Callout warning={true}>
```

Comment:
Is there a link we can point customers to on how to do that? maybe from the Cognito docs?

Reply:
Yes, this looks sufficient - https://docs.aws.amazon.com/cognito/latest/developerguide/user-pool-settings-client-apps.html

I'll add and update